skrf/media/media.py

```diff
@@ -1310,6 +1310,55 @@ def shunt_inductor(self, L: NumberLike, **kwargs) -> Network:
         return self.shunt(self.inductor(L=L, **kwargs) **
                           self.short(**kwargs), **kwargs)
 
+    def capacitor_q(self, C: NumberLike, f: NumberLike, q_factor: NumberLike, **kwargs) -> Network:
+        r"""
+        Capacitor with Q factor.
+
+        Parameters
+        ----------
+        C : number, array-like
+            Capacitance in Farads.
+        f : number
+            Frequency at which Q is defined
+        q_factor : number
+            Q-factor of capacitor
+        \*\*kwargs : arguments, keyword arguments
+            passed to func:`capacitor`
+
+        Returns
+        -------
+        capacitor_q : :class:`~skrf.network.Network` object
+            capacitor_q (2-port)
+
+        """
+        idea_cap = self.shunt(self.capacitor(C=C))
+        rac = q_factor / (C * 2 * npy.pi * f)
+        idea_res = self.shunt(self.resistor(R=rac))
+
+        return innerconnect(connect(idea_cap, 1, idea_res, 2), 1, 3)
```

Comment:
couldn't be simpler? like:

```suggestion
        return media.capacitor(C=C) ** media.shunt_resistor(R=rac)
```